vue-app/.env.example

```diff
@@ -11,6 +11,8 @@ VUE_APP_CLRFUND_FACTORY_ADDRESS=0x5FC8d32690cc91D4c39d9d3abcBD16989F875707
 
 # Supported values: simple, brightid
 VUE_APP_USER_REGISTRY_TYPE=simple
+# clr.fund (prod) or CLRFundTest (testing)
+VUE_APP_BRIGHTID_CONTEXT=clr.fund
```

Comment:
Good call! We should also use this in our deploy scripts when we pass context into the registry constructor, e.g. https://github.com/ethereum/clrfund/blob/develop/contracts/scripts/deploy.ts#L45

Reply:
Not sure if we have to encode the string in any way?

Reply:
Yes, I was just starting to look into that. I think we need to encode it, you are right.